sirepo/package_data/template/srw/beamline_optics.py.jinja

```diff
@@ -38,9 +38,13 @@ v.op_{{ item.name }}_{{ nameMap.get(name, name) }}
 {% if loop.index == 1 %}
         if el_name == '{{ item.name }}':
 {% else %}
+        {% if item.type not in ('watch', 'sample') %}
         elif el_name == '{{ item.name }}':
```

Comment:
Only for item.type == 'watch'. The "sample" type is a different beamline element which we do want to include.

Reply:
I know this is confusing because sometimes watchpoints have the name "Sample" which is the position of the sample in the actual beamline.

Reply:
Also try the Example/Sample from Image SRW example to make sure it works normally.

Reply:
It looks like the generated code for the beamlineAnimation model depends on watches being present in names:
```python
        for n in beamline_info.names:
            names.append(n)
            if n in beamline_info.watches:
                #...
```
So there will need to be some care in removing watches from the generated output, but preserving it in the internal data structures used to generated the code.

Reply:
> Also try the Example/Sample from Image SRW example to make sure it works normally.

made the changes. Re the Sample from Image example, i get the error message in run.log:
```

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 628739 RUNNING AT v.radia.run
=   EXIT CODE: 9
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Killed (signal 9)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
```
But I get that error on master too